packages/app/src/visualizer/VisSelector.module.css

```diff
@@ -7,7 +7,7 @@
   composes: btnClean from global;
   display: flex;
   align-items: center;
-  padding: 0.375rem 1rem;
+  padding: 0.125rem 1rem 0;
```

Comment:
Removing the now-obsolete padding top/bottom, but keeping a bit to align tab labels better.